pynopegl-utils/pynopegl_utils/viewer/__init__.py

```diff
@@ -39,6 +40,35 @@
 
 import pynopegl as ngl
 
+_LICENSE_APACHE = "https://github.com/NopeFoundry/nope.gl/blob/main/LICENSE"
+_NOTICE = "https://github.com/NopeFoundry/nope.gl/blob/main/NOTICE"
+_NOPE_FOUNDRY = "https://www.nope-foundry.org"
+_NOPE_MEDIA = "https://github.com/NopeFoundry/nope.media"
+_ABOUT_POPUP_CONTENT = dedent(
+    f"""\
+    # About
+
+    The Nope viewer is part of the [Nope Foundry]({_NOPE_FOUNDRY}) project and
+    is licensed under the [Apache License, Version 2.0]({_LICENSE_APACHE}).
+
+    See the [NOTICE]({_NOTICE}) file for more information.
+
+    What you create with this software is your sole property. All your artwork
+    is free for you to use as you like. That means that the Nope Foundry can be
+    used commercially, for any purpose. There are no restrictions whatsoever.
+
+    This software also relies on [nope.media]({_NOPE_MEDIA}), which is under
+    LGPL v2.1.
+
+    Finally, it depends on the following third-party libraries:
+    - [FFmpeg](https://www.ffmpeg.org): LGPL v2.1 / GPL v3
+    - [Harfbuzz](https://harfbuzz.github.io): MIT
+    - [FriBidi](https://github.com/fribidi/fribidi): LGPL v2.1
+    - [FreeType](https://freetype.org): FTL / GPL v2
+    - [Qt6](https://www.qt.io): GPL v3
```

Comment:
We should add MoltenVK on macOS (even though the viewer does not use it yet, nopegl is linked against it).

Reply:
Added